### ISSUE TITLE
Ability to edit batchsize of Component and Extended mask types

### ISSUE THREAD

**Issue by acdop100** [deepfakes/faceswap#1221]
**Is your feature request related to a problem? Please describe.**
It would be nice if we could change the batch size of the Component and Extended masks for extraction. I am able to run large batch sizes for all of the detectors, aligner, and other masks. 

**Describe the solution you'd like**
Just a way to set a custom batch size in the GUI for those two masks the same as the add-on masks. 

**Describe alternatives you've considered**
I don't think there are any alternatives. I tried looking through the config files manually but nothing was there related to those masks and their batch sizes. 

**Additional context**
Whenever I start an extraction, I see that the components and extended masks always get initialized to batch size of one in the logs.

**Comment by torzdf** [comment]
The reason there isn't a batchsize for this is that the components and extended masks are "dummy" mask plugins. They use the same plugin architecture as the other masks, which are generated through neural networks, but they are actually entirely generated from the extracted landmarks.

With this in mind, it doesn't make sense to have a batchsize for these masks, as they will always be generated as one face = one mask. Increasing the batch size would just increase the number of faces sent to the maskers, but they would still be processed one at a time, so there would be zero difference in processing time.

**Comment by acdop100** [comment]
That makes sense, thanks for the explanation. In that case, would it make sense to allow a user to not run those maskers during extracting if, say, I only intended to train with the bisenet-Fp Face mask? That way I don't have to waste processing power creating masks I don't intend to use. Unless I am misunderstanding the use of those two masks.

**Comment by torzdf** [comment]
Honestly, the overhead is negligible. When I looked to add them as default, I tested extraction speed with and without creating them and there was no measurable difference.

The reason I add them as default is for a couple of reasons...

1) To make sure that end users **always** have a mask available. For new users, this is crucial, as without a mask available the final swap will look bad. When people are learning the process, then these masks being available is very valuable. Later, of course, you realize that there are better options available. Also, the default training settings have "Penalized Mask Loss" enabled with a default of "Extended" mask. This ensures that new users are not immediately hit by an error when they start training.

2) The components mask is required for the eye/mouth multiplier during training, as they tell the Neural network where the eyes and mouth are located.
